src/vs/workbench/contrib/preferences/browser/preferencesWidgets.ts

```diff
@@ -267,12 +267,12 @@ export class SettingsTargetsWidget extends Widget {
 		this.userLocalSettings = new Action('userSettings', '', '.settings-tab', true, () => this.updateTarget(ConfigurationTarget.USER_LOCAL));
 		this.preferencesService.getEditableSettingsURI(ConfigurationTarget.USER_LOCAL).then(uri => {
 			// Don't wait to create UI on resolving remote
-			this.userLocalSettings.tooltip = uri?.fsPath ?? '';
+			this.userLocalSettings.tooltip = localize('userSettings', "User");
 		});
 
 		this.userRemoteSettings = new Action('userSettingsRemote', '', '.settings-tab', true, () => this.updateTarget(ConfigurationTarget.USER_REMOTE));
 		this.preferencesService.getEditableSettingsURI(ConfigurationTarget.USER_REMOTE).then(uri => {
-			this.userRemoteSettings.tooltip = uri?.fsPath ?? '';
+			this.userRemoteSettings.tooltip = localize('userSettings', "User");
```

Comment:
The remote settings tab usually says something different. To see it, try opening the Settings editor after connecting to a remote using the Remote-SSH, Dev Containers, or Remote-WSL extension.

Reply:
It seems that it does still display the path when you hover the mouse over a remote workspace.


![Screenshot 2023-06-09 183049](https://github.com/microsoft/vscode/assets/46573438/db40ec24-0054-4a0d-9206-b5f8c55ddf61)

Reply:
Oh sorry about that i get what you mean know. Would it be fair to only display Remote or Remote with the connection (ex: Remote[SSH ...])

Reply:
Sorry for the delayed response, here. I'd prefer remote with the connection. That way, a screenreader user doesn't lose information relative to a non-screenreader user.